djangobulk/bulk.py

```diff
@@ -10,18 +10,41 @@
 from django.db import models, connections, transaction
 
 
+def _model_keys(model, field_names=None):
+    """Takes a model class and returns a list of fields that should be
+       used in a WHERE clause to an UPDATE.
+
+    :param model: A model class.
+    :param field_names: An iterable of field names that should be returned if
+        they match field in the model. If empty, no fields are returned.
+        If None the model's primary key is returned.
+    :returns: A list of fields that should be used in a WHERE
+    """
+    # If no fields are specified, use the primary key by default
+    if field_names is None:
+        return [model._meta.pk.name]
```

Comment:
`model_keys` should be instances of `django.db.models.fields`, so `if field_names is None` then `[model._meta.pk]` should be returned instead